pages/platform/ai/deploy_tuto_09_streamlit_speech_to_text_app/guide.en-gb.md

```diff
@@ -0,0 +1,246 @@
+---
+title: AI Deploy - Tutorial - Create and deploy a Speech to Text application using Streamlit
+slug: deploy/tuto-streamlit-speech-to-text-app
+excerpt: How to create and deploy a Streamlit Application for Speech To Text 
+section: AI Deploy - Tutorials
+order: 09
+---
+
+**Last updated 2nd December, 2022.**
+
+> [!primary]
+>
+> AI Deploy is in `beta`. During the beta-testing phase, the infrastructure’s availability and data longevity are not guaranteed. Please do not use this service for applications that are in production, as this phase is not complete.
+>
+> AI Deploy is covered by **[OVHcloud Public Cloud Special Conditions](https://storage.gra.cloud.ovh.net/v1/AUTH_325716a587c64897acbef9a4a4726e38/contracts/d2a208c-Conditions_particulieres_OVH_Stack-WE-9.0.pdf)**.
+>
+
+![Overview](images/speech-to-text.png){.thumbnail}
+
+## Objective
+
+The purpose of this documentation is to **Deploy the Speech to Text Application** we have realised in the blog articles using [Streamlit](https://streamlit.io/) and pre-trained models. 
```

Comment:
Hello @MathieuBsqt , could you please add a link to the mentioned blog article ?

Reply:
Hi @OVH-Team-Guides , thanks, I just corrected this. 
I couldn't do it before because the blog post wasn't out yet.